### PR TITLE
release: 3.6.4

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### Feat:
* Create a separate package.json for CommonJS (cjs) module during build time. This package.json is located under dist/cjs directory. (#687)
* Add a new prop `isUserIdUsedForNickname` to the public interface. This prop allows using the userId as the nickname. (#683)
* Add an option to the ChannelProvider: `reconnectOnIdle`(default: true), which prevents data refresh in the background. (#690)

### Fixes:
* Fix an issue where the server returns 32 messages even when requesting 31 messages in the Channel. Now, hasMorePrev will not be set to false when the result size is larger than the query. (#688)
* Verify the fetched message list size with the requested size of the MessageListParams. Added a test case for verifying the fetched message list size. (#686)
* Address the incorrect cjs path in package.json. The common js module path in the pacakge.json has been fixed. (#685)

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
I'll rebase again if there's more commit to include.

**Comment by sendbird-sdk-deployment** [comment]
[Creating Ticket] 🔖 Done https://sendbird.atlassian.net/browse/SDKRLSD-913

**Review by sravan-s — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog - v3
 
+## [v3.6.4] (July 19 2023)
+### Feat:
+* Created a separate package.json for CommonJS (cjs) module during build time. This package.json is located under dist/cjs directory. (#687)
```
Comment (HoonBaek) [inline CHANGELOG.md:5]: Can we use the present tense for consistency?

Comment (AhyoungRyu) [reply CHANGELOG.md]: Yeah updated it https://github.com/sendbird/sendbird-uikit-react/pull/689/commits/612d30e9b25ce77d76162cfef6a832ad30d284f2

**Comment by AhyoungRyu** [comment]
I added one more commit #690

**Pull request merged**